packages/db/prisma/schema.prisma

```diff
@@ -8,11 +8,28 @@ datasource db {
 
 generator client {
   provider        = "prisma-client"
-  output          = "../src/generated"
+  output          = "../src/generated/prisma"
   previewFeatures = ["fullTextSearchPostgres"]
   engineType      = "client"
 }
 
+generator zod {
+  provider = "prisma-zod-generator"
+  output   = "../src/generated/zod"
+}
```

Comment:
what's the purpose of this actually? do we have use cases where this is required? Most types are already inferred properly yeah? I am wary of this mostly due to latency that may be added due to having zod. if no real use case yet maybe don't need.

Reply:
this creates zod schemas from the models. i added this in (albeit with minimal config) because a non-negligible portion of work is in setting up crud endpoints with the corresponding validator - this makes that boilerplate work less troublesome in the future!